requirements/edx/base.txt

```diff
@@ -99,15 +99,14 @@ edx-bulk-grades==0.6.6    # via -r requirements/edx/base.in, staff-graded-xblock
 edx-ccx-keys==1.0.0       # via -r requirements/edx/base.in
 edx-celeryutils==0.3.2    # via -r requirements/edx/base.in, super-csv
 edx-completion==3.1.1     # via -r requirements/edx/base.in
-edx-django-oauth2-provider==1.3.5  # via -r requirements/edx/base.in, edx-oauth2-provider, edx-organizations, edxval
+edx-django-oauth2-provider==1.3.5  # via edx-organizations, edxval
```

Comment:
Guess this is still referenced from edx-organizations and edxval. Do you want to handle that in a separate PR first?

Reply:
Thanks for catching this. We're looking into removing the usages of this from edx-organizations and edx-val now. I'll also take a look at the cookiecutter to make sure we're not still including this.

Reply:
I think it would be good to get this removed first before merging, to ensure that nothing is actually relying on this.  Other than that, looks good to go.

Reply:
Yup, that's the plan. Get those fixes in, rerun `make upgrade` then verify the tests are still clean.